ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro.yaml

```diff
@@ -62,13 +52,10 @@ tests:
     cluster_profile: azure-qe
     test:
     - as: set-upgrade-releases
-      commands: echo "${RELEASE_IMAGE_INTERMEDIATE411},${RELEASE_IMAGE_INTERMEDIATE412},${RELEASE_IMAGE_TARGET}"
-        > ${SHARED_DIR}/upgrade-edge
+      commands: echo "${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
       dependencies:
-      - env: RELEASE_IMAGE_INTERMEDIATE411
-        name: release:intermediate411
-      - env: RELEASE_IMAGE_INTERMEDIATE412
-        name: release:intermediate412
+      - env: ARO_CLUSTER_VERSION
```

Comment:
What the expected upgrade version path?

If it is a z-stream upgrade, or one-hop upgrade, I do not think we need to `set-upgrade-releases` step.

`set-upgrade-releases`  is only used in multiple hops upgrade (aka chain upgrade) case.

And I do not think this env is being used correctly as an item of `dependencies`, an item in `dependencies` need to be pointed to a release image, but not some version string.